gravitl/netclient
# Gra 999/proxy cmd

Review After PR #140 is merged

Testing:
1. use cmd `netclient proxy on` to turn on the proxy
    

- [x]    verify the status proxy on UI
- [x]    on the host check whether peer endpoint set to loop address (i.e in range of 127.0.0.1/24)
- [x]    check wireguard connections are working

2. use cmd `netclient proxy off` to turn off proxy
     
- [x]   verify the status proxy on UI
- [x]   all peers endpoint should be reset to their actual public endpoints
- [x]   check wireguard connections are working
